### PR TITLE
feature/dropdown-watchlist

### PR DESCRIPTION
**Pull request by DmitriVanGuard**
#### Summary
- Displaying `Watchlist` submenu in `Assets` menu;
- Showing only top 3 watchlists;
- Showing the number of assets in the watchlist.

#### Screenshots
image

### CODE REVIEW

File: app/src/components/DesktopMenu/DesktopWatchlistSubmenu.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+export default graphql(WatchlistGQL, {
+  name: 'Watchlists',
+  options: ({ isLoggedIn }) => ({
+    skip: !isLoggedIn
+  }),
+  props: ({ Watchlists }) => {
+    const { fetchUserLists = [] } = Watchlists
+    return {
+      watchlists: fetchUserLists.slice(0, 3)
```
Comment (Partysun) [inline app/src/components/DesktopMenu/DesktopWatchlistSubmenu.js]: @DmitriVanGuard Could you create a constant for 3?

File: app/src/components/DesktopMenu/DesktopWatchlistSubmenu.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        </Link>
+      ))}
+    </Fragment>
+  )
+}
+
+export default graphql(WatchlistGQL, {
+  name: 'Watchlists',
+  options: ({ isLoggedIn }) => ({
+    skip: !isLoggedIn
```
Comment (Partysun) [inline app/src/components/DesktopMenu/DesktopWatchlistSubmenu.js]: @DmitriVanGuard Skip has to move out of the options. The structure is {name, props, skip, options}

File: app/src/components/DesktopMenu/DesktopWatchlistSubmenu.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import { graphql } from 'react-apollo'
+import { WatchlistGQL } from '../WatchlistPopup/WatchlistGQL'
+
+import styles from './DesktopProfileMenu.module.css'
+
+DesktopWatchlistSubmenu.defaultProps = {
+  watchlists: []
+}
+
+function DesktopWatchlistSubmenu ({ watchlists }) {
```
Comment (Partysun) [inline app/src/components/DesktopMenu/DesktopWatchlistSubmenu.js]: Why did you use a function word here? @DmitriVanGuard

File: app/src/components/DesktopMenu/DesktopProfileMenu.module.css [hunk 4]
```diff
@@ -47,6 +50,11 @@
   align-items: center;
 }
 
+.amount {
+  float: right;
```
Comment (Partysun) [inline app/src/components/DesktopMenu/DesktopProfileMenu.module.css]: @DmitriVanGuard Do we really need to use float here? Maybe we can replace it with flexbox?

File: app/src/components/DesktopMenu/DesktopWatchlistSubmenu.js (hunk 3 above)
Comment (DmitriVanGuard) [reply app/src/components/DesktopMenu/DesktopWatchlistSubmenu.js]: Misused snippet. Should be an arrow function.

File: app/src/components/DesktopMenu/DesktopProfileMenu.module.css (hunk 4 above)
Comment (DmitriVanGuard) [reply app/src/components/DesktopMenu/DesktopProfileMenu.module.css]: Yep, I guess it will be better to use the flexbox here.

**Pull request merged**